packages/mobile/src/alert/AlertBanner.test.tsx

```diff
@@ -1,16 +1,43 @@
 import * as React from 'react'
 import 'react-native'
-import * as renderer from 'react-test-renderer'
+import { render } from 'react-native-testing-library'
 import { AlertBanner } from 'src/alert/AlertBanner'
 
+/*
+import SmartTopAlert, { NotificationTypes } from '@celo/react-components/components/SmartTopAlert'
```

Comment:
Remove dead code here? Or are you planning to restore this?

Reply:
Thanks, removed!